transport/response.hh

```diff
@@ -82,6 +88,7 @@ public:
     void write_string_list(std::vector<sstring> string_list);
     void write_bytes(bytes b);
     void write_short_bytes(bytes b);
+    void write_bytes_map(std::map<sstring, bytes> bytes_map);
```

Comment:
I know `write_bytes_map` is sticking to the same definition as other `write_` functions, but wouldn't it be better if those functions had a const reference as an argument (especially `write_string_list`, `write_bytes`, etc.)? Or am I missing something?

cc @haaawk @jul-stas